src/c++/perf_analyzer/test_custom_load_manager.cc

```diff
@@ -33,28 +33,165 @@
 #include "custom_load_manager.h"
 #include "doctest.h"
 #include "request_rate_manager.h"
+#include "test_load_manager_base.h"
+
+using std::chrono::nanoseconds;
```

Comment:
Small nit: can you rewrite this as:

```
using nanoseconds = std::chrono::nanoseconds
```

and similarly for similar cases?

Reply:
I'll include it in my next story